src/libertem/win_tweaks.py

```diff
@@ -2,10 +2,21 @@
 import msvcrt
 import ctypes
 from ctypes import windll, wintypes, byref
+import logging
 
 import win32security
 import pywintypes
 
+logger = logging.getLogger(__name__)
+logger.setLevel(logging.ERROR)
+
+formatter = logging.Formatter('%(levelname)s : %(message)s : %(asctime)s')
+
+file_handler = logging.FileHandler('win_tweaks.log')
+file_handler.setFormatter(formatter)
+
+logger.addHandler(file_handler)
+
```

Comment:
Is there a reason you are adding a file logger here? IMHO logging configuration should be done in one central place.

Reply:
I thought that if someone faces some problem in the future it would make their job easier to pinpoint the issue.